examples/motoko_examples/superheroes/README.md

```diff
@@ -1,3 +1,63 @@
 # Superheroes
 
-Reimplementation of the [Motoko Superheroes example](https://github.com/dfinity/examples/tree/master/motoko/superheroes)
\ No newline at end of file
+Reimplementation of the [Motoko Superheroes example](https://github.com/dfinity/examples/tree/master/motoko/superheroes)
+
+This example demonstrates how to build a
```

Comment:
If you could keep the README consistent with the other examples, let's not copy in the README yet, just leave it as it was.